src/test_l1_dsVideoPort.c

```diff
@@ -381,19 +397,19 @@ void test_l1_dsVideoPort_negative_dsGetVideoPort(void) {
 	// Step 03: Attempt to get the Video Port handle with invalid port type
 	for (int i = 0; i < NUM_OF_PORTS; i++) {
 		status = dsGetVideoPort(dsVIDEOPORT_TYPE_MAX, kPorts[i].id.index, &(handle[i]));
-		DS_ASSERT_AUTO_TERM_NUMERICAL(status, dsERR_OPERATION_NOT_SUPPORTED);
+		CHECK_FOR_EXTENDED_ERROR_CODE(status, dsERR_OPERATION_NOT_SUPPORTED, dsERR_INVALID_PARAM);
```

Comment:
API is already returning dsERR_INVALID_PARAM which is expected behaviour. change 
DS_ASSERT_AUTO_TERM_NUMERICAL(status, dsERR_OPERATION_NOT_SUPPORTED);  to

UT_ASSERT_EQUAL(status, dsERR_INVALID_PARAM);

Reply:
done.